docs/docs/getting-started/installation/02-licensing.md

```diff
@@ -33,65 +35,44 @@ If an unauthorized user has gained access to your license key, contact support@a
 :::
 
-Additionally, an AICA System License can come in two variants: **online** and **offline**.
-
-## Online licenses
-
 :::info
 
-- Online licenses require an active internet connection for the entire duration of the application.
-- Online licenses can be used on any machine, but only one instance can be running at any one time.
+- AICA System Licenses are online and require an active internet connection for the entire duration of the application.
+- An AICA System License can be used on any machine, but only one instance can be running at any one time.
 
 :::
 
-An online license will appear in the following format, though the specific license key will be unique for each user.
+An AICA System License will appear in the following format, though the specific license key will be unique for each
+user.
 
-```toml title="aica-license.toml"
-License = "5614D1-3E7A6C-932DEB-8C4189-F6B0F2-V3"
+```console title="Example AICA System License"
+5614D1-3E7A6C-932DEB-8C4189-F6B0F2-V3
 ```
 
-## Offline licenses
-
-:::info
-
-- Offline licenses can be used without an active internet connection.
-- Offline licenses must be registered and activated on a specific machine, and can only be used on that machine.
-- Offline licenses are a premium service and are available on request for customers with strict network limitations or
-  requirements.
+## Deployment Key
 
-:::
+A Deployment Key authorizes a specific **deployment** to run offline with no limitations on consecutive or total
+activation time. A **deployment** is a single machine (i.e., an industrial edge PC) installed with some configuration
+of the AICA System.
 
-To prepare a target machine for an offline license, run the following command to identify its unique machine
-fingerprint.
+A Deployment Key will appear as a very long string of characters in the following format, though the specific key will
+be unique for each machine.
 
-```shell
-docker run --privileged --rm ghcr.io/aica-technology/machine-fingerprint
+```console title="Example AICA Deployment Key"
+key/eyJ9df2jfap7IVdIHnlnNpb2482Hh2193L9io8[...]ExMWM3jAuMCJ9.YhsDjwEz8eYnwE21alSBR_tBSIjavblcziV5nBQ==
 ```
 
-If it is not possible to temporarily provide network access to the target machine, first pull and save
-the `ghcr.io/aica-technology/machine-fingerprint` utility docker image on an online machine, then transfer and load it
-on the target machine:
+### Requesting a Deployment Key
 
-```shell
-# download the machine fingerprint helper an online machine
-user@online-machine:~$ docker pull ghcr.io/aica-technology/machine-fingerprint
-user@online-machine:~$ docker save ghcr.io/aica-technology/machine-fingerprint > machine-fingerprint.tar
+Deployment Keys can be requested by AICA System License holders by using AICA Launcher v1.3.2 or newer on the target
+machine. Press the green stamp icon under a selected configuration generate a unique machine fingerprint and follow
+the instructions to send the activation code to the AICA support team.
```

Comment:
Above, we use contact@aica.tech and now support@aica.tech. Is this the desired workflow?